### ISSUE TITLE
rules_webtesting is failing with Bazel@HEAD in CI

### ISSUE THREAD

**Issue by sgowroji** [bazelbuild/rules_webtesting#458]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/3165#01895229-4fb7-4ff9-9027-04b0cb372a34

Platform : Ubuntu, MacOS, Windows
Logs : 
```
return java_common.pack_sources(
	File "/virtual_builtins_bzl/common/java/java_common.bzl", line 150, column 68, in _pack_sources
Error in _check_java_toolchain_is_declared_on_rule: Rule 'scala_library_for_plugin_bootstrapping' in '/private/var/tmp/_bazel_buildkite/babcd828cef764fd2f88fc3cf799c018/external/io_bazel_rules_scala/third_party/dependency_analyzer/src/main/BUILD:6:39' must declare '@bazel_tools//tools/jdk:toolchain_type' toolchain in order to use java_common.
(04:24:42) ERROR: /private/var/tmp/_bazel_buildkite/babcd828cef764fd2f88fc3cf799c018/external/io_bazel_rules_scala/third_party/dependency_analyzer/src/main/BUILD:6:39: Analysis of target '@io_bazel_rules_scala//third_party/dependency_analyzer/src/main:scala_version' failed
(04:24:42) ERROR: Analysis of target '//scalatests/com/google/testing/web:WebTestTest_chrome-win10-connect' failed; build aborted: Analysis

```

Steps:
```
git clone https://github.com/bazelbuild/rules_webtesting.git 
git reset 9f39b9dd292c763cef44cc63719e399373b17631 --hard
export USE_BAZEL_VERSION=53914865fc26850a1f282a83f3846c6103af217e
bazel build //...

```

CC Greenteam @SalmaSamy

**Comment by sgowroji** [comment]
Hi @DrMarcII , Can you please have a look on this issue. Thanks!

**Comment by sgowroji** [comment]
Hello @DrMarcII or the rules\_webtesting team, in case the failures persist without resolution for an extended period, we may need to transition this to the disabled pipeline. It would be greatly appreciated if you could respond and provide us with an update. Thank you!

**Comment by DrMarcII** [comment]
I am no longer working on web testing in any form. I have messaged someone
who might care about this to see what they want to do.

**Comment by mtrea** [comment]
What's the implication of transitioning to the disabled pipeline? Do you have any idea what the suggested fix is for this issue or what change might be causing it?

**Comment by sgowroji** [comment]
CC @meteorcloudy

**Comment by meteorcloudy** [comment]
> What's the implication of transitioning to the disabled pipeline?

@sgowroji meant the downstream pipeline for Bazel@HEAD, check the doc and policy here: https://github.com/bazelbuild/continuous-integration/blob/master/docs/downstream-testing.md

I believe this is the same root cause as https://github.com/bazelbuild/rules_jvm_external/issues/931, but from the error message, it seems this should be fixed in rules_scala first.

**Comment by sgowroji** [comment]
Oh thats fine. I will wait for rules_jvm_external to fix this first. @mtrea  Sorry for the remainder, please ignore it.

**Comment by meteorcloudy** [comment]
@sgowroji No, I meant those two have the same root cause, but should be fixed in both projects. That means a similar change should be made in rules_webtesting (or in rules_scala then update rules_scala in rules_webtesting).
